unitTests/ui/ResultsPerPageTest.ts

```diff
@@ -20,9 +21,30 @@ export function ResultsPerPageTest() {
       test = null;
     });
 
-    it('should trigger a query when the number of results per page changes', () => {
-      test.cmp.setResultsPerPage(50);
-      expect(test.env.queryController.executeQuery).toHaveBeenCalled();
+    describe('when calling #setResultsPerPage', () => {
```

Comment:
Not sure it's 100% needed, but since the initial reported problem was that multiple results per page component in the same page we're not "synchronizing" properly, would it be a good idea to add a smoke test for this ?

Even though in theory it should/will work with those changes...

Reply:
Yes, it's a good idea. I will add one.